Make syncing laboratory directory clearer before running ACCESS-OM (#703)
ACCESS-NRI/access-hive.org.au: docs/models/run-a-model/run-access-om.md

This design allows multiple self-resubmitting experiments that share common executables and input data to be run simultaneously.

!!! admonition warning

>     The _laboratory_ directory might get deleted if not accessed for several days. To know more about how to preserve your experiment refer to [Syncing output data](#syncing-output-data).

Reviewer: is "accessed" enough? Doesn't it get deleted a few months after the last edit?

Author: [NCI website reference](https://opus.nci.org.au/plugins/servlet/mobile?contentId=156434436#content/view/156434436) says "accessed". That's why I wrote that.
Not sure how the deletion rule actually works though.

>     The _laboratory_ directory might get deleted if not accessed for several days. To know more about how to preserve your experiment refer to [Syncing output data](#syncing-output-data).

Reviewer: ```suggestion
    For all model runs which are being kept for any length of time, or being analysed, best practice is to enable the _sync_ step in Payu. The _laboratory_ directory might get deleted if not accessed for several days and the `/scratch` folder for some projects is limited in space. To know more about how to preserve your experiment refer to [Syncing output data](#syncing-output-data).
```

Author: I updated the text by revising your suggestion.

> 

Reviewer: ```suggestion
The ACCESS-OM2 default configurations include a [userscript](#userscript) in the _sync_ step in Payu. The script concatenates daily history/diagnostic output from the Sea-Ice model (CICE5) into one file per month. This speeds up access and saves storage space but will only run if sync is enabled.
```

Author: I would add this bit to the sync option paragraph rather than here in the admonition.

>     The _laboratory_ directory might get deleted if not accessed for several days, and the `/scratch` folder is often limited in space. For these reasons, all model runs which are to be kept for more than a few months should be preserved by enabling the _sync_ step in _payu_. To know more refer to [Syncing output data](#syncing-output-data).

Reviewer: ```suggestion
    Files on the `/scratch` drive, such as the _laboratory_ directory, might get deleted if not accessed for several days and the `/scratch` drive can limited in space. For these reasons, all model runs which are to be kept  should be moved to `/g/data/ by enabling the _sync_ step in _payu_. To know more refer to [Syncing output data](#syncing-output-data).
```

      - '*.nc.*'
      - 'iceh.????-??-??.nc'
```

> To enable syncing, change `enable` to `True`, and set `path` to a location on `/g/data`, where _payu_ will copy output and restart folders.<br>

Reviewer: ```suggestion
To enable syncing, change `enable` to `True`, and set `path` to a location on `/g/data`, where _payu_ will copy output and restart folders. 
```

> For example, a sensible `path` could be: `/g/data/$PROJECT/$USER/experiment_name/`.

Reviewer: ```suggestion
A sensible `path` could be: `/g/data/$PROJECT/$USER/experiment_name/`.
```

!!! admonition tip

>     The {{model}} default configurations include a [userscript](#userscripts) in the _sync_ step that concatenates daily history/diagnostic output from the Sea-Ice model (CICE5) into monthly files. This speeds up access and saves storage space, and will only run if _sync_ is enabled.

Reviewer: ```suggestion
    The {{model}} default configurations include a [userscript](#userscripts) in the _sync_ step that concatenates daily history/diagnostic output from the seaice model (CICE5) into monthly files. This speeds up access and saves storage space but will only run if _sync_ is enabled.
```
